api-reference/docstubs/resources/enums.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "Enum values"
```

Comment:
What was the reason that the enums.md file was generated?

Reply:
Good point - not sure we need to do this.